pkg/printers/humanreadable_flags.go

```diff
@@ -28,13 +28,14 @@ import (
 type HumanPrintFlags struct {
 	ShowKind     *bool
 	ShowLabels   *bool
-	ColumnLabels *[]string
 	SortBy       *string
+	ColumnLabels *[]string
+
+	// get.go-specific values
+	NoHeaders bool
```

Comment:
How should we best differentiate values that are command-specific, and can be bound to a flag, vs values that are not set via a flag?

Reply:
Comment is the only one that comes to my mind.